demo/.env

```diff
@@ -1 +1,4 @@
-REACT_APP_GA_TRACKING_ID=G-70CMQK5VQE
\ No newline at end of file
+REACT_APP_GTAG_TRACKING_ID=G-70CMQK5VQE
+REACT_APP_GTAG_API_SECRET=e45cwxLtQ46v5dLqr9v5dw
```

Comment:
gtag doesn't use the API secret for transmission (it uses a different wire protocol that doesn't require one), so this could be removed from the environment.  Probably not much harm in leaving it, but just thinking on the principle of not putting more into this then we need to.

Reply:
Ah, I didn't notice that it didn't need the api secret, that's interesting. I removed the line.